alphametics.json

```diff
@@ -21,6 +21,16 @@
                 "description": "solve puzzle with exponents",
                 "puzzle": "PI * R ^ 2 == AREA",
                 "expected": { "P": 9, "I": 6, "R": 7, "A": 4, "E": 0 }
+            },
+            {
+                "description": "invalid solution returns null",
```

Comment:
You are thinking right in that it could go anywhere, the communication is not exactly pinned down yet for the titles.

About the null, yes, this is true, but JSON is used in all languages using this, and so... is there a probl... yes, the person reading from the exercism site, not seeing or realizing the tie-in with JSON... never mind. :)

I like your title suggestion, can it be just as descriptive, but more concise?

'solutions must have unique value for each letter' which naturally restricts to 10 letters, in a base 10 numbering system?